src/libcollections/fmt.rs

```diff
@@ -261,7 +261,9 @@
 //! This and `writeln` are two macros which are used to emit the format string
 //! to a specified stream. This is used to prevent intermediate allocations of
 //! format strings and instead directly write the output. Under the hood, this
-//! function is actually invoking the `write` function defined in this module.
+//! function is actually invoking the
+//! [`write_fmt`](../../io/trait.Write.html#method.write_fmt) function defined on the
```

Comment:
It looks like the paths are wrong. I've tried `../io/trait.Write.html` and `../../io/trait.Write.html` but neither seem to work. @GuillaumeGomez any ideas?